scripts/closure-compiler/tsconfig-rxjs.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Yeah would be nice, but unfortunately RxJS doesn't ship their `tsconfig.json`. We only have the plain sources from the NPM package. See [here](https://unpkg.com/rxjs@5.2.0/src/)